packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -407,14 +407,14 @@ class _Element extends _Entry {
     return children.firstWhere(
         (_Entry e) => e is _Attribute && e.key.startsWith(name),
         orElse: () => null,
-    );
+    ) as _Attribute;
```

Comment:
you can factor out the `e is _Attribute` using `whereType` and that'll get rid of the `as`

Reply:
Done, here and below